### PR TITLE
feature/VDX-290

### PR DESCRIPTION
**Pull request by BtencateSphereon**

### CODE REVIEW

File: packages/oid4vci-demo-frontend/public/locales/development/en/translation.json [hunk 1]
```diff
@@ -90,7 +90,7 @@
   "paragraph_scan_this_qr": "Scan this QR code using your wallet's QR scanner. When you already on your phone open your wallet by clicking on the Open in wallet button.",
   "paragraph_confirm_email": "Confirm your account by entering the 5 digit confirmation code from the email.",
   "loading_verify_company_information_caption": "Verifying the company information",
-  "loading_verify_identity_information_caption": "Verifying your identity.",
+  "loading_verify_identity_information_caption": "Verifying your identity",
```
Comment (sanderPostma) [inline packages/oid4vci-demo-frontend/public/locales/development/en/translation.json:93]: Dutch?

File: packages/oid4vci-demo-frontend/src/pages/SSIIdentityVerificationPage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+const SSIIdentityVerificationPage: React.FC = (): ReactElement => {
+    const {t} = useTranslation()
+    const flowRouter = useFlowRouter<SSIIdentityVerificationPageConfig>()
+    const pageConfig = flowRouter.getPageConfig();
+    const isTabletOrMobile = useMediaQuery({query: '(max-width: 767px)'})
+    const location = useLocation();
+    const state = location.state
+
+    const onContinue = async (): Promise<void> => {
+        await flowRouter.nextStep({payload: state})
```
Comment (sanderPostma) [inline packages/oid4vci-demo-frontend/src/pages/SSIIdentityVerificationPage/index.tsx]: No longer required, you can also maybe remove lines 17 & 18 if not used anywhere else

**Review by sanderPostma — CHANGES_REQUESTED** [review]
One minor issue. One optional
Approved when the Dutch translation line is added

File: packages/oid4vci-demo-frontend/public/locales/development/en/translation.json (hunk 1 above)
Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/public/locales/development/en/translation.json:93]: i only removed the . dutch translation is there without the .

File: packages/oid4vci-demo-frontend/src/pages/SSIIdentityVerificationPage/index.tsx (hunk 2 above)
Comment (BtencateSphereon) [reply packages/oid4vci-demo-frontend/src/pages/SSIIdentityVerificationPage/index.tsx]: fixed

**Review by sanderPostma — APPROVED** [review]

**Pull request merged**